README.md

```diff
@@ -151,6 +151,7 @@ A curated list of awesome niche job boards.
 * [Python Job Board](https://www.python.org/jobs/)
 * [Django Jobs](https://djangojobs.net/jobs/)
 * [Python Developer Jobs](https://pythonjob.xyz)
+[Python Devs](https://www.pythondevs.net)
```

Comment:
Hey @s892307 , this line is missing the leading asterisk (`*`) to denote the list entry.

Also, is this even a job board? I don't see any jobs listed.

Reply:
Doh, thanks @tramcarjobs!  I've added the asterisk now.

It's a reverse jobs board, so devs list their profile and companies can reach out to them.  I think it's relevant to your list as it helps devs find work :)